attack.py

```diff
@@ -4890,7 +4892,20 @@ def pre_starting_info():
         ||||
         ||||
 """)
-
+def killnon_critical():
+    for proc in psutil.process_iter(['pid', 'name', 'cpu_percent']):
+        try:
+            process_info = proc.info
+            print(process_info)
+            
+            # Kill processes using minimal CPU (This is risky and just for demonstration)
+            if process_info['cpu_percent'] < 1:  # Assume non-critical if low CPU usage
```

Comment:
ERROR:
'<' not supported between instances of 'NoneType' and 'int'

Reply:
got the reason of error will look into it